# HG changeset patch
# User Rik <rik@octave.org>
# Date 1454723904 28800
#      Fri Feb 05 17:58:24 2016 -0800
# Node ID 88b41a4711e2bb043de105aaf3fac9fa8f578ddd
# Parent  fcac5dbbf9ed76ac10cade200d47c39acfff80a0
maint: Correct names used in #ifdefs to prevent re-loading .h files.

* randgamma.h: Correct "octave_radgamma" to "octave_randgamma".

* randmtzig.h: Correct "octave__randmtzig_h" to "octave_randmtzig_h".

* oct-uname.h: Correct "octave_uname_h" to "octave_oct_uname_h".

diff --git a/liboctave/numeric/randgamma.h b/liboctave/numeric/randgamma.h
--- a/liboctave/numeric/randgamma.h
+++ b/liboctave/numeric/randgamma.h
@@ -19,17 +19,17 @@ along with Octave; see the file COPYING.
 <http://www.gnu.org/licenses/>.
 
 */
 
 /* Original version written by Paul Kienzle distributed as free
    software in the in the public domain.  */
 
 #if ! defined (octave_randgamma_h)
-#define octave_radgamma_h 1
+#define octave_randgamma_h 1
 
 #ifdef  __cplusplus
 extern "C" {
 #endif
 
 extern OCTAVE_API double oct_randg (double a);
 extern OCTAVE_API void oct_fill_randg (double a, octave_idx_type n, double *p);
 
diff --git a/liboctave/numeric/randmtzig.h b/liboctave/numeric/randmtzig.h
--- a/liboctave/numeric/randmtzig.h
+++ b/liboctave/numeric/randmtzig.h
@@ -57,21 +57,21 @@ along with Octave; see the file COPYING.
    PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
    LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
    NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
    SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 */
 
 #if !defined (octave_randmtzig_h)
-#define octave__randmtzig_h 1
+#define octave_randmtzig_h 1
 
 #define MT_N 624
 
-#ifdef  __cplusplus
+#ifdef __cplusplus
 extern "C" {
 #endif
 
 /* Mersenne Twister.  */
 extern OCTAVE_API void oct_init_by_int (uint32_t s);
 extern OCTAVE_API void oct_init_by_array (uint32_t *init_key, int key_length);
 extern OCTAVE_API void oct_init_by_entropy (void);
 extern OCTAVE_API void oct_set_state (uint32_t *save);
diff --git a/liboctave/system/oct-uname.h b/liboctave/system/oct-uname.h
--- a/liboctave/system/oct-uname.h
+++ b/liboctave/system/oct-uname.h
@@ -16,17 +16,17 @@ for more details.
 
 You should have received a copy of the GNU General Public License
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
 #if ! defined (octave_oct_uname_h)
-#define octave_uname_h 1
+#define octave_oct_uname_h 1
 
 #include <string>
 
 class
 OCTAVE_API
 octave_uname
 {
 public:
